src/main/java/dev/openfeature/sdk/providers/memory/InMemoryProvider.java

```diff
@@ -0,0 +1,184 @@
+package dev.openfeature.sdk.providers.memory;
```

Comment:
I think this is a good package name and differentiate from contrib provider implemenations (`dev.openfeature.contrib.providers.*`)